core/extension/gdextension_interface.h

```diff
@@ -1383,6 +1384,18 @@ typedef GDExtensionVariantFromTypeConstructorFunc (*GDExtensionInterfaceGetVaria
  */
 typedef GDExtensionTypeFromVariantConstructorFunc (*GDExtensionInterfaceGetVariantToTypeConstructor)(GDExtensionVariantType p_type);
 
+/**
+ * @name get_variant_get_internal_ptr_func
```

Comment:
There's a whole bunch of `variant_get_ptr_*()` functions that return function pointers. Maybe this should just be `variant_get_ptr_internal`?

That would match `variant_get_ptr_constructor`, `variant_get_ptr_destructor`, `variant_get_ptr_operator_evaluator`, etc, that all return function pointers.

Reply:
Makes sense to me!

Reply:
It looks like the name was updated everywhere, except right here :-)

```suggestion
 * @name get_variant_get_ptr_internal
```

Reply:
Er, I thought we agreed on `variant_get_ptr_internal`? Your latest push is using a totally new name

Using `variant_get_ptr_internal` would match all the other `Variant`-related functions that return function pointers, for example, `variant_get_ptr_constructor`, `variant_get_ptr_destructor`, `variant_get_ptr_operator_evaluator`, etc

Reply:
Doesn't the `@name` have to match the `REGISTER_INTERFACE_FUNC` call? I was going off of your suggestion from above with `@name get_variant_get_ptr_internal`

Reply:
Ah, sorry, I type-o'd on my suggestion from ~20 minutes ago. I intended to put the name of the function you were already using at that time and which we agreed on earlier in this thread, which was `variant_get_ptr_internal`

Reply:
Cool, then it should be correct now :)

Reply:
Looks good, thanks!